src/nanoemoji/colr_to_svg.py

```diff
@@ -267,20 +267,39 @@ def descend(parent: etree.Element, paint: otTables.Paint):
         raise NotImplementedError(ot_paint.Format)
 
 
+def glyph_region(ttfont: ttLib.TTFont, glyph_name: str) -> Rect:
+    return Rect(
+        0,
+        ttfont["OS/2"].sTypoDescender,
```

Comment:
SVG goes y-down (not y-up), so the glyph_region rectangle (in font-space) that you want to map to the viewbox in SVG space also needs to go y-down, like below.
Once you apply a patch like that, you'll notice that the <g> color glyphs of the SVG created by maximum_color from a COLR input font will no longer have any `transform` attribute, because the coordinates are already in font-space and there's a 1:1 mapping between the spaces.
Also, the baseline will now correctly be at y=0 in both OT-SVG and glyf.

```diff
diff --git a/src/nanoemoji/colr_to_svg.py b/src/nanoemoji/colr_to_svg.py
index 78dbda6..a657eaa 100644
--- a/src/nanoemoji/colr_to_svg.py
+++ b/src/nanoemoji/colr_to_svg.py
@@ -57,8 +57,9 @@ ViewboxCallback = Callable[[str], Rect]  # f(glyph_name) -> Rect
 
 
 def map_font_space_to_viewbox(view_box: Rect, glyph_region: Rect) -> Affine2D:
-    ascender = glyph_region.h + glyph_region.y
-    descender = glyph_region.y
+    ascender = -glyph_region.y
+    assert ascender >= 0
+    descender = ascender - glyph_region.h
     assert descender <= 0
     width = glyph_region.w
 
@@ -270,7 +271,7 @@ def _colr_v1_paint_to_svg(
 def glyph_region(ttfont: ttLib.TTFont, glyph_name: str) -> Rect:
     return Rect(
         0,
-        ttfont["OS/2"].sTypoDescender,
+        -ttfont["OS/2"].sTypoAscender,
         ttfont["hmtx"][glyph_name][0],
         ttfont["OS/2"].sTypoAscender - ttfont["OS/2"].sTypoDescender,
     )
diff --git a/src/nanoemoji/svg.py b/src/nanoemoji/svg.py
index 0756985..23fbfb7 100644
--- a/src/nanoemoji/svg.py
+++ b/src/nanoemoji/svg.py
@@ -441,7 +441,9 @@ def _add_glyph(svg: SVG, color_glyph: ColorGlyph, reuse_cache: ReuseCache):
         raise ValueError(f"{color_glyph.svg_filename} must declare view box")
 
     # https://github.com/googlefonts/nanoemoji/issues/58: group needs transform
-    svg_g.attrib["transform"] = _svg_matrix(color_glyph.transform_for_otsvg_space())
+    transform = color_glyph.transform_for_otsvg_space()
+    if not transform.almost_equals(Affine2D.identity()):
+        svg_g.attrib["transform"] = _svg_matrix(transform)
 
     vbox_to_upem = color_glyph.transform_for_font_space()
     upem_to_vbox = vbox_to_upem.inverse()

```

Reply:
y-down. Must remember y-down!